apps/manager_app/manager_app.c

```diff
@@ -129,5 +132,8 @@ void manager_app_main(usb_event_t usb_evt) {
     }
   }
 
+  // TODO: Check if on-boarding default screen is to be rendered
+  onboarding_set_static_screen();
```

Comment:
is this the right place to reset flow status and device state as follows?
```
  core_status_set_flow_status(0);
  core_status_set_idle_state(CORE_DEVICE_IDLE_STATE_DEVICE_IDLE_STATE_IDLE);
```

Reply:
there refers to the application (here `manager_app`), right?

Reply:
Added both at appropriate places